UI/src/model/selectedLaneModelling.js

```diff
@@ -74,44 +74,39 @@
         return numberOfLanesByLaneCode[key] > 1;
       });
 
-      var lanesSortedByLaneCode = _.sortBy(lanes, getLaneCodeValue);
-
-      var duplicateLaneCounter = 0;
-      return _.map(lanesSortedByLaneCode, function (lane) {
-        if(_.includes(laneCodesToPutMarkers, getLaneCodeValue(lane).toString())) {
-          if (duplicateLaneCounter === 0){
-            lane.marker = 'A';
-            duplicateLaneCounter++;
-          }else{
-            lane.marker = 'B';
-            duplicateLaneCounter--;
-          }
-        }
-        return lane;
+      var lanesSortedByEndMeasure = _.sortBy(lanes, function(lane) {
+        return lane.endMeasure;
       });
+
+      var charCount = 0;
```

Comment:
Mikä tämä merkkaa ?

Reply:
Se on siis laskuri tota kaistanpätkän kirjaintunnusta varten. Eli A:lla se on nolla, ja siitä eteenpäin.

Reply:
ehkä kuuvampi muuttujan nimi voisi olla paikalla.